recipes/openturns/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "1.7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: openturns
+  version: {{ version }}
+
+source:
+  fn: openturns-{{ version }}.tar.gz
+  url: https://github.com/openturns/openturns/archive/v{{ version }}.tar.gz
+  sha1: 9535120721b20a45460538f3ed8fa0f0fec1d797
+
+build:
+  number: 200
+  skip: true  # [py3k or win or osx]
+  features:
+    - blas_{{ variant }}
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - python
+    - matplotlib
+    - cmake
+    - swig
+    - boost
+    - gcc  # [linux]
```

Comment:
Not actually. This can be easily avoided. See this [line]( https://github.com/conda-forge/staged-recipes/pull/578/files#diff-209088c915d08aae01cca3fdc25af178R29 ). Really we should enforce using the `toolchain` in all cases to avoid this sort of confusion. I plan to start doing just that.